includes/database.php

```diff
@@ -0,0 +1,82 @@
+<?php
+
+namespace Wporg\TranslationEvents;
+
+use Exception;
+use WP_Query;
+use Wporg\TranslationEvents\Attendee\Attendee;
+
+class Database {
+	public static function upgrade(): void {
+		require_once ABSPATH . 'wp-admin/includes/upgrade.php';
+		global $gp_table_prefix;
+
+		// Create actions table.
+		$create_actions_table = "
+		CREATE TABLE `{$gp_table_prefix}event_actions` (
+			`translate_event_actions_id` int(11) unsigned NOT NULL AUTO_INCREMENT,
+			`event_id` int(10) NOT NULL COMMENT 'Post_ID of the translation_event post in the wp_posts table',
+			`original_id` int(10) NOT NULL COMMENT 'ID of the translation',
+			`user_id` int(10) NOT NULL COMMENT 'ID of the user who made the action',
+			`action` enum('approve','create','reject','request_changes') NOT NULL COMMENT 'The action that the user made (create, reject, etc)',
+			`locale` varchar(10) CHARACTER SET latin1 COLLATE latin1_swedish_ci NOT NULL COMMENT 'Locale of the translation',
+			`happened_at` datetime NOT NULL COMMENT 'When the action happened, in UTC',
+		PRIMARY KEY (`translate_event_actions_id`),
+		UNIQUE KEY `event_per_translated_original_per_user` (`event_id`,`locale`,`original_id`,`user_id`)
+		) COMMENT='Tracks translation actions that happened during a translation event'";
+		dbDelta( $create_actions_table );
+
+		// Create attendees table.
+		$create_attendees_table = "
+		CREATE TABLE `{$gp_table_prefix}event_attendees` (
+			`translate_event_attendees_id` int(11) unsigned NOT NULL AUTO_INCREMENT,
```

Comment:
Why we are using the table name in the field name and not only the `id` word? If we change it, we should change the `PRIMARY KEY` field.

```suggestion
			`id` int(11) unsigned NOT NULL AUTO_INCREMENT,
```

Reply:
This was done for consistency with the `event_actions` table, where the field is named `translate_event_actions_id`. I believe we named the primary key like that after feedback we received from wordpress.org maintainers, so I figured it would make sense to follow the same naming pattern here.

In fact, the primary key is technically not needed, but since the `event_actions` table has it (and it's also not technically needed), I guessed I should probably add it here too.